package.json

```diff
@@ -35,7 +35,11 @@
     "swagger-express-mw": "^0.6.0",
     "os-tmpdir": "~1.0.1",
     "rimraf": "~2.5.0",
-    "tar": "~2.2.1"
+    "tar": "~2.2.1",
+    "jsonwebtoken": "^5.4.4",
+    "passport": "^0.3.2",
+    "passport-local": "^1.0.0",
+    "passport-jwt": "^1.2.1"
```

Comment:
Not a formal convention, but previously, we keep writing the module name by alphabetical order. So change this or not is OK for me.